rskj-core/src/main/java/co/rsk/peg/bitcoin/InvalidOutpointValueException.java

```diff
@@ -0,0 +1,12 @@
+package co.rsk.peg.bitcoin;
+
+public class InvalidOutpointValueException extends RuntimeException{
```

Comment:
nit: add the extra space before the bracket
```suggestion
public class InvalidOutpointValueException extends RuntimeException {
```

Reply:
It's not nit, it is necessary. I reformatted the whole file, so it will get fixed.